asset.go

```diff
@@ -33,23 +30,17 @@ func (a *Asset) parse() error {
 	if err != nil {
 		return err
 	}
-	file, err := os.OpenFile(assetPath, os.O_RDONLY, 0666)
-	if err != nil {
-		return err
+	assetFile, errAsset := ioutil.ReadFile(assetPath)
+	if errAsset != nil {
+		return errAsset
 	}
-	assetReader := bufio.NewReader(file)
-	for {
-		_line, _, err := assetReader.ReadLine()
-		if err == io.EOF {
-			break
-		}
-		line := string(_line)
-		var prefix string
-		if a.assetType == AssetJavascript {
-			prefix = "//= require "
-		} else {
-			prefix = "/*= require "
-		}
+	var prefix string
+	if a.assetType == AssetJavascript {
+		prefix = "//= require "
+	} else {
+		prefix = "/*= require "
+	}
+	for _, line := range strings.Split(string(assetFile), "\n") {
```

Comment:
Code is simpler in the new version but the previous version is more optimized since you don't need to read the whole file if the prefix exists in the first few lines.